nix/postgres.nix

```diff
@@ -44,6 +44,37 @@ in
           default = "./data/${config.name}";
           description = "The DB data directory";
         };
+
+        pgHbaConf =
+          let
+            mkStringOption = lib.mkOption { type = lib.types.str; };
+            pgHbaConfSubmodules = lib.types.submodule {
```

Comment:
The naming here is plural, but the value it points to is singular (one submodule).